share/create_db.pl

```diff
@@ -0,0 +1,13 @@
+use strict;
```

Comment:
I suggest the following to make it an executable script (and the bits have to be set).
```
#!/usr/bin/env perl
use strict;
```

Reply:
done